code/modules/mining/equipment/monster_organs/rush_gland.dm

```diff
@@ -0,0 +1,86 @@
+/**
+ * On use in hand, makes you run really fast for 5 seconds and ignore injury movement decrease.
+ * On use when implanted, run for longer and ignore all negative movement. Automatically triggers if health is low (to escape).
+ */
+/obj/item/organ/internal/monster_core/reusable/rush_gland
+	name = "rush gland"
+	icon_state = "lobster_gland"
+	icon_state_preserved = "lobster_gland_stable"
+	icon_state_inert = "lobster_gland_decayed"
+	desc = "A lobstrosity's engorged adrenal gland. You can squeeze it to get a rush of energy on demand."
+	desc_preserved = "A lobstrosity's engorged adrenal gland. It is preserved, allowing you to use it for a burst of speed whenever you need it."
+	desc_inert = "A lobstrosity's adrenal gland. It is all shrivelled up."
+	user_status = /datum/status_effect/lobster_rush
+
+#define HEALTH_DANGER_ZONE 30
```

Comment:
put defines at the file header and undef at footer, nesting it between procs is basically pointless.